src/styles/components/_home.scss

```diff
@@ -66,7 +70,8 @@ $breakpoint-top-section-scale-up: 70rem;
         display: flex;
         flex-direction: column;
         justify-content: center;
-        flex-basis: 45%;
+        flex: 0 0 60%; // max width will be 60%
+        max-width: 60%;
```

Comment:
Are these doing the same thing? Just for consistency, since we used the `flex-basis` in the `&-top` selector, should we continue to use `flex` here and get rid of `max-width`?

Reply:
I removed the max-width and it didn't change anything, so it was just extraneous stuff. Thanks for the suggestion!